README.md

```diff
@@ -1,11 +1,136 @@
+[![Unit-tests](https://github.com/facebookresearch/multimodal/actions/workflows/unit_test.yaml/badge.svg)](https://github.com/facebookresearch/multimodal/actions/workflows/unit_test.yaml)
+[![Python version](https://img.shields.io/pypi/pyversions/torchmultimodal-nightly.svg)](https://www.python.org/downloads/)
+[![Downloads](https://static.pepy.tech/personalized-badge/torchmultimodal-nightly?period=total&units=international_system&left_color=blue&right_color=orange&left_text=Downloads%20(nightly))](https://pepy.tech/project/torchmultimodal-nightly)
+
 # TorchMultimodal (Beta Release)
 
+[**Models**](#models) | [**Example scripts**](#example-scripts) | [**Getting started**](#getting-started) | [**Code overview**](#code-overview) | [**Installation**](#installation) | [**Contributing**](#contributing) | [**License**](#license)
+
 ## Introduction
-TorchMultimodal is a PyTorch library for training state-of-the-art multimodal multi-task models at scale. It provides:
+**TorchMultimodal** is a PyTorch library for training state-of-the-art multimodal multi-task models at scale, including both content understanding and generative models. TorchMultimodal contains:
 - A repository of modular and composable building blocks (models, fusion layers, loss functions, datasets and utilities).
```

Comment:
nit, but usually a list should be at least three or structured differently. It might be worth breaking out the full models  (e.g. CLIP, FLAVA) vs the building blocks as they're different offerings.